profiles/common/modules/features/ec_embedded_image/ec_embedded_image.behat.inc

```diff
@@ -86,34 +86,19 @@ class ECEmbeddedImageSubContext extends DrupalSubContextBase {
       ]
     );
 
-    $this->contentTypeWasSetUp = TRUE;
+    $this->permissionWasSetUp = TRUE;
   }
 
   /**
    * Removes the test_media_gallery content type and related permissions.
    *
    * @AfterScenario
    */
-  public function removeMediaGalleryContentType() {
-    if (!$this->contentTypeWasSetUp) {
+  public function resetMediaGalleryPermission() {
+    if (!$this->permissionWasSetUp) {
       return;
     }
 
-    $this->contentTypeWasSetUp = FALSE;
```

Comment:
Ah ok now I understand @voidtek.
Indeed, if we run the method createMediaGalleryContentType one time $testPermissionsSetUp will be true in all subsequent AfterScenarios and method resetMediaGalleryPermission will not return in first test.

@gillesdeudon can you put back "$this->contentTypeWasSetUp = FALSE;"? It is important to be there.